src/components/Tournament/TournamentInfo.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+
+const TournamentInfo = ({id, title, author}) => {
```

Comment:
Further in tournament info we'll have time period and maybe we'll have some livecycle methods in this component. But it just maybe)

Reply:
When we'll need for life cycle methods, we'll add them and convert this component to class =)

Reply:
Okey) I've got your point) I agree.